app.py

```diff
@@ -21,4 +26,19 @@ async def add_role(member: discord.Member, role_id: int):
     await member.add_roles(role)
 
 
+async def set_welcome_message():
+    channel: discord.TextChannel = bot.get_channel(876672670903717948)
+    message = await channel.fetch_message(880000637998338079)
+
+    with open("./docs/welcome.md") as welcome_message_file:
+        welcome_message = welcome_message_file.read()
+        current_md5 = hashlib.md5(message.content.strip().encode()).hexdigest()
+        message_md5 = hashlib.md5(welcome_message.strip().encode()).hexdigest()
+        if current_md5 != message_md5:
+            welcome_message = welcome_message.replace(
+                "$last_updated", str(datetime.date.today())
+            )
+            await message.edit(content=welcome_message, suppress=True)
```

Comment:
This comparison is needed so that we don't send the exact same welcome message everytime Lacus is started from any machine. This could probably use some work if there is a better method.